model/textparse/openmetricsparse_test.go

```diff
@@ -992,3 +993,46 @@ go_gc_duration_seconds_created`)
 	require.Equal(t, "go_gc_duration_seconds", string(copyParser.l.b[copyParser.offsets[0]:copyParser.offsets[1]]))
 	require.False(t, copyParser.skipCTSeries)
 }
+
+func BenchmarkOMParse(b *testing.B) {
+	for parserName, parser := range map[string]func([]byte, *labels.SymbolTable) Parser{
+		"openmetrics": func(b []byte, st *labels.SymbolTable) Parser {
+			return NewOpenMetricsParser(b, st)
+		},
+		"openmetrics-skip-ct": func(b []byte, st *labels.SymbolTable) Parser {
+			return NewOpenMetricsParser(b, st, WithOMParserCTSeriesSkipped())
+		},
+	} {
+		f, err := os.Open("omtestdata.txt")
+		require.NoError(b, err)
+		defer f.Close()
+
+		buf, err := io.ReadAll(f)
+		require.NoError(b, err)
+
+		b.Run(parserName+"/parse-ct/"+"omtestdata.txt", func(b *testing.B) {
```

Comment:
Is this test really different than `BenchmarkParse`? can we create one that makes sense?

Reply:
In this one we're benchmarking just the performance of calling p.CreatedTimestamp on each line of a "scrape". This was my understanding of the requirements

Is there anything else I should consider?

Reply:
I mean it would be nice to have one benchmark for all of those, for consistency. Why we benchmark Created Timestamp for OM but not for Prometheus proto?

Reply:
.. but not a biggie for this PR